src/email-templates/external/campaign.created.hbs

```diff
@@ -1,3 +1,3 @@
 Campaign "{{ campaign.name }}" was created.
 
-Click here to submit your creatives: {{ link-to account.uri "r/collect" campaign.hash }}
+Click here to submit your creatives: {{ link-to account.uri "app" advertiser.pushId "campaigns" campaign.pushId "materials" }}
```

Comment:
Where does this ultimately link to?

Reply:
The material collect form, e.g; https://allured.native-x.parameter1.com/app/2nsdlfuh123ej/campaigns/123n7fuhse/materials